Archive/Imo/Imo1959Q1.lean

```diff
@@ -0,0 +1,41 @@
+/-
+Copyright (c) 2020 Kevin Lacker. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kevin Lacker
+
+! This file was ported from Lean 3 source module imo.imo1959_q1
+! leanprover-community/mathlib commit 5f25c089cb34db4db112556f23c50d12da81b297
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Tactic.Ring
+import Mathlib.Data.Nat.Prime
+
+/-!
+# IMO 1959 Q1
+
+Prove that the fraction `(21n+4)/(14n+3)` is irreducible for every natural number `n`.
+
+Since Lean doesn't have a concept of "irreducible fractions" per se, we just formalize this
+as saying the numerator and denominator are relatively prime.
+-/
+
+
+open Nat
+
+namespace IMO1959Q1
```

Comment:
```suggestion
namespace Imo1959Q1
```
or `Imo1959_q1`, I guess.

We should be consistent with the other PRs, which are presumably going to use the default capitalisation that mathport produces.

We can do a mass cleanup afterwards.